scripts/remove_ledger.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import shutil
+from sys import argv
+script, file_path = argv
+from indy_common.config_util import getConfig
+from indy_common.config_helper import ConfigHelper
+
+config = getConfig()
+config_helper = ConfigHelper(config)
+
+def main(file_path):
```

Comment:
Let's rename `file_parth` to `ledger_name`. What do you think? It's not a path actually.